packages/design-tokens/src/tokens/functional/components/video-player/base.json

```
@@ -0,0 +1,129 @@
+{
+  "brand": {
+    "VideoPlayer": {
+      "border": {
+        "radius": {
+          "value": "var(--brand-borderRadius-medium)"
+        }
+      },
+      "controls": {
+        "gap": {
+          "value": "15px"
```

Does this have to be 15px? Could we use rems, which scale better? If so, can we switch to use..

```suggestion
          "value": "var(--base-size-16)"
```

---

This has been updated in the following [SHA](https://github.com/primer/brand/pull/373/commits/6f9a9c8740844ac6d50be622f1c9aab106c564dc)